UI templates created

Darthrighteous opened the pull request:
created 6 pages namely: homepage, new question, sign in, sign up, profile, question.

Billmike commented on newpost.html:
```diff
@@ -0,0 +1,65 @@
+	
+</body>
+
+</html>
```
Add a new line here

Darthrighteous replied:
the new line should be added where exactly? after line 65?

Darthrighteous replied:
done

Billmike commented on profile.html:
```diff
@@ -0,0 +1,131 @@
+<script src="script.js"></script>
+</body>
+
+</html>
```
Add a new line here

Darthrighteous replied:
done

Billmike commented on question.html:
```diff
@@ -0,0 +1,164 @@
+<script src="script.js"></script>
+</body>
+
+</html>
```
Add a new line here

Darthrighteous replied:
done

Billmike commented on signin.html:
```diff
@@ -0,0 +1,49 @@
+	
+</body>
+
+</html>
```
Add a new line here

Billmike commented:
39 commits? Not acceptable man. The maximum number of commits you are allowed to have is 2. You want to squash those commits into one whole commit. Refer to [this](https://github.com/k88hudson/git-flight-rules) on how to go about squashing commits. Also, your HTML file isn't properly formatted. You can make use of a plugin called Emmet for that.

Billmike commented:
<img width="1440" alt="screen shot 2018-08-21 at 7 44 50 pm" src="https://user-images.githubusercontent.com/24798364/44422627-99852380-a57b-11e8-977e-bb3f9175356d.png">

<img width="1440" alt="screen shot 2018-08-21 at 7 44 57 pm" src="https://user-images.githubusercontent.com/24798364/44422628-99852380-a57b-11e8-926f-8de69e651fcf.png">

The card is still too close to the navbar. It should come down a little. Also, the width of the input fields should be increased. Preferably, as wide as [these are](http://collectui.com/designers/marcinmania/sign-up) within the defined body

Darthrighteous commented:
@Billmike Thanks for the review.
 I have sqaushed commits, added new lines at EOF, adjusted the width of the sign in form container and the inputs fields. 
Let me know if i missed anything!
